pkg/distro/defs/loader_test.go

```diff
@@ -841,9 +841,9 @@ func TestDistrosLoadingExact(t *testing.T) {
 	restore := defs.MockDataFS(baseDir)
 	defer restore()
 
-	distro, err := defs.NewDistroYAML("fedora-43")
+	dist, err := defs.NewDistroYAML("fedora-43")
 	require.NoError(t, err)
-	assert.Equal(t, &defs.DistroYAML{
+	assert.Equal(t, dist, &defs.DistroYAML{
```

Comment:
Nitpick: technically, the `dist` should be the third argument to `Equal()`, because the second is the "expected" value and the third is the "actual" value.

Reply:
Nice catch! Thank you, I will fix this in a followup (usually I get this right but sometimes I forget :)